portal-db/sdk/go/README.md

```diff
@@ -0,0 +1,201 @@
+# Portal DB Go SDK
+
+This Go SDK provides a type-safe client for the Portal DB API, generated using [oapi-codegen](https://github.com/oapi-codegen/oapi-codegen).
+
+## Installation
+
+```bash
+go get github.com/buildwithgrove/path/portal-db/sdk/go
+```
+
+## Quick Start
+
+```go
```

Comment:
Can you provide a functional main file example that points to localhost and makes it work?